src/point_one/fusion_engine/messages/gnss_corrections.h

```diff
@@ -57,9 +57,9 @@ struct alignas(4) LBandFrameMessage : public MessagePayload {
   float doppler_hz = 0;
 
   /**
-   * A pointer to the beginning of the demodulated L-Band frame data.
+   * The beginning of the demodulated L-Band frame data.
    */
-  uint8_t data_payload[0];
+  // uint8_t data_payload[0];
```

Comment:
Is this change intentional?

Reply:
It's intentional. This has come up before, but technically, zero size struct members aren't allowed in the by the C++ standard. They happen to be supported by GCC and clang. There's no ISO standard way to do this sort of thing unfortunately. It's convenient to have, but I've given up trying to support it for cross compilers.

Reply:
We have had an internal ticket to remove these for a while now for similar reasons, so I agree this is the right change.